### PR TITLE
Acceptance doc for saved launch-time configurations

### PR DESCRIPTION
**Pull request by AlanCoding**
Related to https://github.com/ansible/awx/issues/169

### CODE REVIEW

File: docs/prompting.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+inside of `credential`, `vault_credential`, and `extra_credentials`. Providing
+a credential that requirements password input from the user on launch is
+allowed, and the password must be provided along-side the credential, of course.
+
+If the job is being spawned using a saved launch configuration, however,
+all non-machine credential types are managed by a many-to-many relationship
+called `credentials` relative to the launch configuration object.
+When the job is spawned, the credentials in that relationship will be
+sorted into the job's many-to-many credential fields according to their
+type (cloud vs. vault).
```
Comment (AlanCoding) [inline docs/prompting.md:55]: @ryanpetrello

File: docs/prompting.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+##### Standard Pattern with Character Fields
+
+ - `ask_<variable>_on_launch` allows use of
+   - `<variable>`
+
+The standard pattern applies to fields
+
+ - `job_type`
+ - `job_tags`
```
Comment (AlanCoding) [inline docs/prompting.md:19]: `job_tags` is an exception because it maps to `ask_tags_on_launch`

File: docs/prompting.md (hunk 1 above)
Comment (ryanpetrello) [reply docs/prompting.md:55]: Yep, this seems reasonable 👍

File: docs/prompting.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+ - RBAC rules for prompted resources still apply
+
+Those same rules apply when created a schedule from the
+`/api/v2/schedule_job/` endpoint.
+
+#### Credential Password Prompting Restrictions
+
+If a job template uses a credential that is configured to prompt for a
+password at launch, these passwords cannot be saved for later as part
+of a save launch-time configuration. This is for security reasons.
```
Comment (ryanpetrello) [inline docs/prompting.md:121]: 👍

**Comment by AlanCoding** [comment]
merged into branch https://github.com/ansible/awx/compare/devel...AlanCoding:saved_launch_configs?expand=1

I wanted to merge these smaller PRs in order to keep the feature in better sync with `devel`, but without approval I can't do that.

**Pull request closed without merging**